news.xml

```diff
@@ -590,7 +590,7 @@ $ nix-store -qR /run/current-system | grep openssl
       potentially destabilising changes that sometimes occur on the
       unstable branch. You can get NixOS 13.10 ISOs and VirtualBox
       appliances from the <a href="/download.html">download
-        page</a>. See the <a href="https://nixos.org/nix-dev/2013-October/011941.html">announcement</a>
```

Comment:
it redirects to https://releases.nixos.org/nix-dev/2013-October/011941.html

not sure if that is a stable url